test/e2e/tools.mjs

```diff
@@ -74,40 +185,31 @@ export async function getDatasetItems(url) {
 }
 
 /**
- * @param {string | URL} url
+ * @param {string} dirName
  */
-export async function initialize(url) {
-    process.env.APIFY_LOCAL_STORAGE_DIR = getStorage(url);
+export async function initialize(dirName) {
+    process.env.APIFY_LOCAL_STORAGE_DIR = getStorage(dirName);
     process.env.APIFY_HEADLESS = '1'; // run browser in headless mode (default on platform)
     process.env.APIFY_TOKEN = process.env.APIFY_TOKEN ?? await getApifyToken();
     process.env.APIFY_CONTAINER_URL = process.env.APIFY_CONTAINER_URL ?? 'http://127.0.0.1';
     process.env.APIFY_CONTAINER_PORT = process.env.APIFY_CONTAINER_PORT ?? '8000';
 
-    await purgeLocalStorage();
-    console.log('[init] Storage directory:', process.env.APIFY_LOCAL_STORAGE_DIR);
-}
-
-/**
- * @param {number} timeout
- */
-function waitSync(timeout) {
-    const now = Date.now();
+    process.env.STORAGE_IMPLEMENTATION = process.env.STORAGE_IMPLEMENTATION || 'LOCAL';
```

Comment:
```suggestion
    process.env.STORAGE_IMPLEMENTATION ??= LOCAL';
```

Reply:
just noticed when was merging conflicts between master and bacporting, that you removed one `'` sign :) Will fix it in backporting PR